iModelJsNodeAddon/api_package/ts/src/test/DgnDb.test.ts

```diff
@@ -591,20 +610,21 @@ describe("basic tests", () => {
     db.saveChanges();
 
     // import a schema with changes that only required SchemaLock
-    let rc = db.importXmlSchemas([generateSchema(20, 1)], { schemaLockHeld: false });
-    assert.equal(rc, DbResult.BE_SQLITE_OK);
+    db.importXmlSchemas([generateSchema(20, 1)], { schemaLockHeld: false });
     db.saveChanges();
 
     // eslint-disable-next-line @typescript-eslint/naming-convention
     const BE_SQLITE_ERROR_DataTransformRequired = (DbResult.BE_SQLITE_IOERR | 23 << 24);
 
     // import should fail when schemaLockHeld flag is set to false which will fail the operation if data transform is required.
-    rc = db.importXmlSchemas([generateSchema(20, 20)], { schemaLockHeld: false });
-    assert.equal(rc, BE_SQLITE_ERROR_DataTransformRequired);
+    try {
+      db.importXmlSchemas([generateSchema(20, 20)], { schemaLockHeld: false });
+    } catch (error: any) {
+      assert.equal(error.errorNumber, BE_SQLITE_ERROR_DataTransformRequired);

```

Comment:
As mention before if exception is not thrown it will still pass.

Reply:
Updated the test